web/src/popover_menus.js

```diff
@@ -793,11 +793,7 @@ export function initialize() {
                 // emoji_picker which we don't want to hide after actions popover is hidden.
                 e.stopPropagation();
                 e.preventDefault();
-                emoji_picker.toggle_emoji_popover(
-                    instance.reference.parentElement,
-                    message_id,
-                    true,
-                );
+                emoji_picker.show_emoji_popover(instance.reference.parentElement, message_id);
                 instance.hide();
             });
```

Comment:
So this change in naming makes me worry that clicking the emoji popover opening button twice will not hide it.

Reply:
That does seem to work fine -- it's just a built-in feature of the new library. So I think we need to:
* Call the function that you're using here `emoji_picker.toggle_emoji_popover` -- basically it looks like you merged those two functions but kept the wrong name.
* Also name your new helper function `popover_menus.show_popover_menu` instead `popover_menus.toggle_popover_menu` so that it's very clear what its expected behavior is.

Reply:
Yeah, these points are reasonable. I changed the names of the functions to the ones you proposed. Thanks!